coursemapper-kg/concept-map/src/data.py

```
@@ -1,12 +1,13 @@
 import re
 from typing import List, Callable
 from numpy import ndarray
+from typing import Optional
```

Ruff (0.8.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)